recipe/build.sh

```diff
@@ -19,5 +21,46 @@ export INSTALL_INCLUDE="${PREFIX}/include"
 export BLAS="-lopenblas"
 export LAPACK="-lopenblas"
 
+# build METIS first, installing into $PREFIX rather than the source DIR
+cd metis-5.1.0
+make config shared=1 prefix=${PREFIX}
+make
+make install
+
+if [ "$(uname)" == "Darwin" ]
+then
+    # this is copied from the SuiteSparse Makefile
+    install_name_tool -id ${PREFIX}/lib/libmetis.dylib ${PREFIX}/lib/libmetis.dylib
```

Comment:
not sure if the `install_name_tool` call is necessary.  I copied it verbatim from the SuiteSparse Makefile.  I can try without and see if maybe it gets handled for us by conda build.